src/autoscaler/metricsforwarder/forwarder/syslog_emitter.go

```diff
@@ -0,0 +1,102 @@
+package forwarder
+
+import (
+	"fmt"
+	"net/url"
+	"os"
+	"time"
+
+	"code.cloudfoundry.org/app-autoscaler/src/autoscaler/metricsforwarder/config"
+	"code.cloudfoundry.org/app-autoscaler/src/autoscaler/models"
+	"code.cloudfoundry.org/go-loggregator/v9/rpc/loggregator_v2"
+	"code.cloudfoundry.org/lager/v3"
+	"code.cloudfoundry.org/loggregator-agent-release/src/pkg/egress"
+	"code.cloudfoundry.org/loggregator-agent-release/src/pkg/egress/syslog"
+)
+
+type SyslogEmitter struct {
+	logger lager.Logger
+	Writer egress.WriteCloser
+}
+
+type Counter struct{}
+
+func (c *Counter) Add(delta float64) {
+}
+func (c *Counter) Set(delta float64) {
+}
+
+func NewSyslogEmitter(logger lager.Logger, conf *config.Config) (MetricForwarder, error) {
+	var writer egress.WriteCloser
+	var protocol string
+
+	tlsConfig, _ := conf.SyslogConfig.TLS.CreateClientConfig()
+
+	netConf := syslog.NetworkTimeoutConfig{
+		WriteTimeout: time.Second,
+		DialTimeout:  100 * time.Millisecond,
+	}
+
+	if conf.SyslogConfig.TLS.CACertFile != "" {
+		protocol = "syslog-tls"
+	} else {
+		protocol = "syslog"
+	}
+
+	url, _ := url.Parse(fmt.Sprintf("%s://%s:%d", protocol, conf.SyslogConfig.ServerAddress, conf.SyslogConfig.Port))
+
+	logger.Info("syslog-emitter-config", lager.Data{"url": url})
```

Comment:
This is just for debugging in case something goes really weird during the URL-parsing, or?

```suggestion
	logger.Debug("using-syslog-url", lager.Data{"url": url})
```